docs/API.md

```diff
@@ -656,16 +689,50 @@ An object with the following properties:
 
 #### `"result"`
 
-A *test id*. 
+A *test id*.
 
-If the test has been run with the same domain name within an interval of 10 mins (hard coded), 
+If the test has been run with the same domain name within an interval of 10 mins (hard coded),
```

Comment:
`mins` -> `minutes`

Reply:
I realize now that you didn't introduce the "mins" wording, but if you make the update anyway while you're at it, I'd appreciate it.